pkg/cmd/lint.go

```diff
@@ -141,6 +165,21 @@ func lintFile(filename string) error {
 	}
 	foundNames[summonObj.Name] = filename
 
+	// Check that the docker image exists
+	if imageTags != nil {
+		var foundImage bool
+		for _, imageTag := range imageTags {
+			if summonObj.Spec.Version == imageTag {
```

Comment:
Instead loop it will good if we use `manifests`  api
```
 GET | /v2/<name>/manifests/<reference> | Manifest | Fetch the manifest identified by name and reference where reference can be a tag or digest. A HEAD request can also be issued to this endpoint to obtain resource information without receiving all data.
```

Reply:
That would require 1 API call for every instance instead of getting list all at once and then checking. We could slightly speed this up by putting the image tags into a map[string]bool though.